spell-check-excluded-files-list

```diff
@@ -1,6 +1,6 @@
 Adyen/Utilities/External/AllRegions.swift
 Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift
 Adyen/Core/Models/CardType.swift
-Demo/Common/IntegrationExamples/AddressLookupProvider.swift
```

Comment:
Not sure if this is necessary
It is nice to know that we don't have typos in Demo code 🤔